lib/po_libc_wrappers.c

```diff
@@ -41,6 +41,7 @@
 #include <stdarg.h>
 #include <stdlib.h>
 #include <unistd.h>
+#include<string.h>
```

Comment:
This `#include` should come after `#include <stdlib.h>`, alphabetically.

Also, there should be a space between `#include` and `<string.h>`.